tests/Credentials/ServiceAccountCredentialsTest.php

```diff
@@ -495,15 +496,16 @@ public function testNoScopeAndNoAuthUri()
         $this->assertNotNull($sa);
 
         $update_metadata = $sa->getUpdateMetadataFunc();
-        $this->assertTrue(is_callable($update_metadata));
+        $this->assertInternalType('callable', $update_metadata);
 
         $actual_metadata = call_user_func($update_metadata,
             $metadata = array('foo' => 'bar'),
             $authUri = null);
         // no access_token is added to the metadata hash
         // but also, no error should be thrown
-        $this->assertTrue(is_array($actual_metadata));
-        $this->assertTrue(
-            !isset($actual_metadata[CredentialsLoader::AUTH_METADATA_KEY]));
+        $this->assertInternalType('array', $actual_metadata);
+        $this->assertFalse(
+            isset($actual_metadata[CredentialsLoader::AUTH_METADATA_KEY])
```

Comment:
Would this be better as 

```php
 $this->assertArrayNotHasKey(CredentialsLoader::AUTH_METADATA_KEY, $actual_metadata);
```

Reply:
Good one! Gonna change it and search for similars